File: Abstraction/C#/Shape.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ConsoleApp2
{
    public abstract class Shape
    {
        public abstract void GetNumberofCorners();
    }

    public class Pentagon : Shape
    {
       /*
        * Method : GetNumberofCorners
        * Output Parameter : Write the output
        */
       public override void GetNumberofCorners()
       {    
            Console.WriteLine("Pentagon has five corners.");
       }

       static void Main(string[] args)
       {
            Pentagon objPentagon = new Pentagon();                
            objPentagon.GetNumberofCorners();
       }
    }
}

File: Abstraction/Shape.py
from abc import ABC, abstractmethod
 
class Shape(ABC):
    
    @abstractmethod
    def GetNumberofCorners(self):
        pass
        
class Pentagon(Shape):
    def GetNumberofCorners(self):
        print("Pentagon has five corners.")
        
x = Pentagon()
x.GetNumberofCorners()